docs/api/index.rst

```diff
@@ -6,9 +6,5 @@ Packages
     :glob:
 
     base/index
-    io/index
```

Comment:
you can't just remove all of these sections from the documentation.
they are there because they should be there!

Reply:
I had added back.